src/puppetlabs/puppetdb/query/events.clj

```diff
@@ -9,6 +9,7 @@
             [puppetlabs.puppetdb.query-eng.engine :as qe]
             [puppetlabs.puppetdb.schema :as pls]
             [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.scf.storage-utils :as scf-utils]
```

Comment:
sutils is used in more files then scf-utils, its a convention schism I know, but I'd use sutils 'cos its shorter and avoids the hyphen :-).